WebUI/war/services/PercServiceUtils.js

```diff
@@ -282,7 +282,7 @@
         makeAjaxRequest(args);
     }
     function makeAjaxRequest(args){
```

Comment:
@ramdev1980  Does this cover all of the ajax calls?  There are a number of methods  on this plugin that I would expect to see toggle the client side timeout.